docs/web-integration.md

```diff
@@ -0,0 +1,88 @@
+# Integrating the assistant
+
+This section is about integrating the assistant in Nextcloud's web frontend.
+
+The assistant can be used anywhere in Nextcloud's frontend. The assistant modal can be opened with or without initial input.
+It can also be opened to see the result of a task.
+
+## Displaying a task result in the assistant
+
+There are 2 ways to display a task with the assistant.
+
+### Open the assistant modal
+
+If you get a task with the `/ocs/v2.php/apps/assistant/api/v1/task/TASK_ID` or `/ocs/v2.php/apps/assistant/api/v1/tasks` OCS endpoint,
+you can display it in the assistant UI by using the `OCA.Assistant.openAssistantTask` helper function.
+This function takes the task object as parameter. Calling it will open the assistant and load the task values
```

Comment:
Maybe we can have a link here for the task object to point to the below section? And mention it explicitly there?